runners/spark/src/main/java/org/apache/beam/runners/spark/translation/SparkStreamingPortablePipelineTranslator.java

```diff
@@ -108,6 +110,9 @@ public SparkStreamingPortablePipelineTranslator() {
     translatorMap.put(
         PTransformTranslation.RESHUFFLE_URN,
         SparkStreamingPortablePipelineTranslator::translateReshuffle);
+    translatorMap.put(
+        "urn:beam:transform:redistribute:v1", // Add the appropriate URN for Redistribute
```

Comment:
there are two urns for Redistribution, PTransformTranslation.REDISTRIBUTE_BY_KEY and PTransformTranslation.REDISTRIBUTE_ARBITRARILY

Reply:
Sure, I just replace it with existing ones
REDISTRIBUTE_BY_KEY_URN and REDISTRIBUTE_ARBITRARILY_URN in PTransformTranslation.java